### PR TITLE
system.events and system.metrics tables add column name as an alias to event and metric

### LINKED ISSUE #51257: system.events and system.metrics tables should have column `name` as an alias to `event` and `metric`

### PR DESCRIPTION
**Pull request by xiedeyantu**
### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
system.events and system.metrics tables add column name as an alias to event and metric. close #51257

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by robot-ch-test-poll3** [comment]
This is an automated comment for commit d0e886070c6acbb57c35a1d3df287e2a07a9f115 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/52315/d0e886070c6acbb57c35a1d3df287e2a07a9f115/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>🟡 pending</td></tr>
</table>

File: src/Storages/System/StorageSystemEvents.cpp [hunk 1]
```diff
@@ -16,6 +16,13 @@ NamesAndTypesList StorageSystemEvents::getNamesAndTypes()
     };
 }
 
+NamesAndAliases StorageSystemEvents::getNamesAndAliases()
+{
+    return {
+        {"name", std::make_shared<DataTypeString>(), "event"}
```
Comment (kssenii) [inline src/Storages/System/StorageSystemEvents.cpp:22]: Let's add a simple stateless test for this.

**Review by kssenii — APPROVED** [review]

**Comment by kssenii** [comment]
Could you please english documentation about this change?

**Comment by xiedeyantu** [comment]
> Could you please english documentation about this change?

done

**Comment by qoega** [comment]
>@kssenii Hi, this pr https://github.com/ClickHouse/ClickHouse/issues/52540 has nothing to do with mine.

We know. Just it is a fuzzer that found interesting case when running in this PR.

**Review by alexey-milovidov — APPROVED** [review]

**Comment by kssenii** [comment]
> Integration tests (release) [2/4] — fail: 1

`test_backup_restore_on_cluster/test_concurrency.py::test_create_or_drop_tables_during_backup[Replicated-ReplicatedMergeTree]`
Issue: https://github.com/ClickHouse/ClickHouse/issues/45437

**Pull request merged**